packages/prism-theme-odyssey/webpack.config.js

```diff
@@ -0,0 +1,31 @@
+var path = require('path');
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+
+module.exports = {
+   devServer: {
+    index: 'index.html',
+    contentBase: path.join(__dirname, 'dist'),
+    compress: true,
+    port: 9000
+  },
+  module: {
+    rules: [
+      {
+        test: /\.s[ac]ss$/i,
+        use: [
+          // Creates `style` nodes from JS strings
+          'style-loader',
+          // Translates CSS into CommonJS
+          'css-loader',
+          // Compiles Sass to CSS
+          'sass-loader',
+        ],
+      },
```

Comment:
> I don't quite follow where are we consuming `sass` and therefore need to use the `sass-loader`. Can you help me understand this better?
> 
>_Originally posted by @jmelberg-okta in https://github.com/okta/odyssey/pull/714#discussion_r509797918_

Reply:
We are importing partials from Odyssey 

https://github.com/okta/odyssey/pull/730/files#diff-d55846964f7ead7d0690796169fb46dfdcf377eac4cdfacd28ba705a5dd1e8a8R1-R3